### DISCUSSION TITLE
Is it possible to refer two schemas under a single property?

### DISCUSSION THREAD

**Discussion by singhayushh** [Automattic/mongoose#10087] [Q&A] [answered]
Hello, 
I want to setup a model for _reviews_ where there is to be a property named _author_ which is to be a reference to one of two existing schemas: Admin or User:
```
author: {
        type: mongoose.Schema.Types.ObjectId, 
        ref: either 'Admin' or 'User'
}
```

I am wondering if this is possible to do.

**Comment by vkarpov15** [comment, marked as answer]
Sure, use [`refPath`](https://mongoosejs.com/docs/populate.html#dynamic-ref)
